src/Jackett.Common/Definitions/rutor.yml

```diff
@@ -51,18 +51,22 @@
   search:
     paths:
       # http://rutor.info/search/0/0/000/0/gotham%2004x01
-      - path: "{{ if .Keywords }}search/0/0/{{ .Config.method }}00/0/{{ .Keywords }}{{else}}top{{end}}"
+      - path: "{{ if .Query.Q }}search/{{ .Query.Q }}{{else}}top{{end}}"
```

Comment:
if you are going to drop the config.method from the path then you might as well remove it from the settings block.